bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh

```diff
@@ -128,6 +129,7 @@ export MATOMO_DATABASE_PORT_NUMBER="${MATOMO_DATABASE_PORT_NUMBER:-3306}" # only
 export MATOMO_DATABASE_NAME="${MATOMO_DATABASE_NAME:-bitnami_matomo}" # only used during the first initialization
 export MATOMO_DATABASE_USER="${MATOMO_DATABASE_USER:-bn_matomo}" # only used during the first initialization
 export MATOMO_DATABASE_PASSWORD="${MATOMO_DATABASE_PASSWORD:-}" # only used during the first initialization
+export MATOMO_DATABASE_TABLE_PREFIX="${MATOMO_DATABASE_TABLE_PREFIX:-}" # only used during the first initialization
```

Comment:
For compatibility reasons, we should keep the default `matomo_` value. Could you change it?

```suggestion
export MATOMO_DATABASE_TABLE_PREFIX="${MATOMO_DATABASE_TABLE_PREFIX:-matomo_}" # only used during the first initialization
```

Reply:
Just pushed a fix with fde647c, apologies for the oversight.

Reply:
It seems to be the same, could you re-check it?

Reply:
Sorry about that, forgot to git add the edited file. Should be there now.